BookingProject/BookingWebAPI/Payment.Domain/Models/UserBalance.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Payment.Domain.Models
+{
+    public class UserBalance
+    {
+        public int UserBalanceId { get; set; }
+        public string UserId { get; set; }
+        public decimal UAH { get; set; }
+        public decimal USD { get; set; }
+        public decimal EUR { get; set; }
```

Comment:
Adding new currency will cause u to add migration. Use json approach or use 1:M table for currencies